Bio/Phylo/TreeConstruction.py

```diff
@@ -374,38 +374,38 @@ class DistanceCalculator(object):
     --------
     >>> from Bio.Phylo.TreeConstruction import DistanceCalculator
     >>> from Bio import AlignIO
-    >>> aln = AlignIO.read(open('Tests/TreeConstruction/msa.phy'), 'phylip')
+    >>> aln = AlignIO.read(open('./TreeConstruction/msa.phy'), 'phylip')
     >>> print(aln)
     SingleLetterAlphabet() alignment with 5 rows and 13 columns
     AACGTGGCCACAT Alpha
     AAGGTCGCCACAC Beta
+    CAGTTCGCCACAA Gamma
     GAGATTTCCGCCT Delta
     GAGATCTCCGCCC Epsilon
-    CAGTTCGCCACAA Gamma
 
     DNA calculator with 'identity' model::
 
         >>> calculator = DistanceCalculator('identity')
         >>> dm = calculator.get_distance(aln)
-        >>> print(dm)
-        Alpha   0
-        Beta    0.230769230769  0
-        Gamma   0.384615384615  0.230769230769  0
-        Delta   0.538461538462  0.538461538462  0.538461538462  0
-        Epsilon 0.615384615385  0.384615384615  0.461538461538  0.153846153846  0
-                Alpha           Beta            Gamma           Delta           Epsilon
+        >>> print(dm) # doctest: +NORMALIZE_WHITESPACE
```

Comment:
Is this change (and the whitespace changes below) needed to pass, or just cosmetic?

Reply:
Yes, they are needed to pass, the output includes tabs; as far as I understand unittest replaces tabs with spaces, so is a bit tricky to make it match the output; tried using the text as r""" and including tabs (\t) but it keep failing; found this online and passed.